README.md

```diff
@@ -893,6 +893,14 @@ May also be configured in `lerna.json`:
 }
 ```
 
+#### --reject-cycles
+
+Fail immediately if a cycle is found (in `oobstrap`, `exec`, `publish` or `run`).
+
```

Comment:
FYI theres a typo in the word "bootstrap" both in the line above and two lines bellow.